### ISSUE TITLE
Project Reunion runtime installer shows provisioning result of E_ACCESSDENIED during install

### ISSUE THREAD

**Issue by riverar** [microsoft/WindowsAppSDK#885]
**Describe the bug**
When attempting to install the Project Reunion runtime, package deployment succeeds but several provisioning steps appear to fail with E_ACCESSDENIED. It's not clear if the runtime installer is required to be elevated.

**Steps to reproduce**
1. .\ProjectReunion-0.8Preview-Install-x64.exe
2. Observe results
```
Deploying package: Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x86__8wekyb3d8bbwe
Package deployment result : 0x0
Deploying package: Microsoft.ProjectReunion.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Package deployment result : 0x0
Deploying package: Microsoft.ProjectReunion.Main.0.8-preview_8000.146.628.0_x64__8wekyb3d8bbwe
Package deployment result : 0x0
Provisioning result : 0x80070005
Deploying package: Microsoft.ProjectReunion.DDLM.8000.146.628.0-x8-p_8000.146.628.0_x86__8wekyb3d8bbwe
Package deployment result : 0x0
Provisioning result : 0x80070005
Deploying package: Microsoft.ProjectReunion.DDLM.8000.146.628.0-x6-p_8000.146.628.0_x64__8wekyb3d8bbwe
Package deployment result : 0x0
Provisioning result : 0x80070005
All packages were installed successfully.
```

ProjectReunion-0.8Preview-Install-x64.exe
Windows 10 21H1 19043.985

**Comment by andrewleader** [comment]
@DrusTheAxe or @AdamBraden can you help with this?

**Comment by DrusTheAxe** [comment]
@dkbennett is Stage tried as a best-effort sort of thing? It's only expected to succeed if the installer's running elevated? If so you can wrap it with

```
#include <Security.IntegrityLevel.h>
…
if (Security::IntegrityLevel()::IsElevated())
{
    ...stage the package...
}
```

and not error in the expected not-expected-to-work case

**Comment by dkbennett** [comment]
Not sure if this is still a thing, but assigning to @DrusTheAxe for re-triage.

**Comment by DrusTheAxe** [comment]
https://aka.ms/38095216

**Comment by sachintaMSFT** [comment]
This is fixed in 1.1 milestone.
